### ISSUE TITLE
Cant Play Due To New Captcha/kisscartoon

### ISSUE THREAD

**Issue by CapnCranch** [Twoure/KissNetwork.bundle#68]
Kissanime has been under attack by a DMCA claims bot and has implemented a captcha bot. So the kissanime part of this channel wont work cause we cant complete the captcha. Kisscartoon was taken down and they have been having people use kisscartoon.

**Comment by Twoure** [comment]
> Kissanime has been under attack by a DMCA claims bot and has implemented a captcha bot. 

Yes, captcha is an issue.  Added mobile site as backup for Anime in https://github.com/Twoure/KissNetwork.bundle/commit/7130447e6d60bb092cb1b292c4a7482cbfc5781d but may not work anymore.  Haven't had time to investigate further.

> Kisscartoon was taken down and they have been having people use kisscartoon.

Refer to https://github.com/Twoure/KissNetwork.bundle/issues/69 for more info.
